backend/services/media_generation_service.py

```
@@ -0,0 +1,421 @@
+        """Generate video placeholder image (for video preparation)"""
+        try:
+            # Create vertical image for TikTok format
+            img = Image.new('RGB', (params['width'], params['height']), params['background_color'])
+            draw = ImageDraw.Draw(img)
+            
+            # Load fonts
+            try:
+                title_font = ImageFont.truetype("arial.ttf", 80)
+                quote_font = ImageFont.truetype("arial.ttf", 50)
+                video_font = ImageFont.truetype("arial.ttf", 35)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)